golang.req

```diff
@@ -21,6 +33,9 @@ end
 requires = []
 
 unless filelist.empty? then
+
+	requires << "golang(API) = " + go_get_version()
```

Comment:
maybe I am stupid. but Linus style of collaboration has one fundamental fact that Linus is one of the major authors of the Kernel. this is not the case. I see little code that convince people. so it sounds to me like an outsider who don't have his hands dirty are pointing here and there without solid evidences and codes. 

Or maybe, please, you can treat me as an idiot who hold the nuclear suitcase. You have to stop her from pressing the wrong button. That needs a lot. Not just a "No".